packages/state-transition/src/util/epochShuffling.ts

```diff
@@ -60,21 +105,14 @@ export function computeCommitteeCount(activeValidatorCount: number): number {
 
 export function computeEpochShuffling(
   state: BeaconStateAllForks,
-  activeIndices: ArrayLike<ValidatorIndex>,
-  activeValidatorCount: number,
+  activeIndices: ValidatorIndex[],
```

Comment:
```suggestion
  activeIndices: Uint32Array,
```

then no need to create `_activeIndices` below, just use the input `activeIndices` which was populated once in `beforeProcessEpoch`
this should save us 1 memory allocation